place_tracker/lib/place_tracker_app.dart

```diff
@@ -18,17 +18,9 @@ class PlaceTrackerApp extends StatefulWidget {
 }
 
 class _PlaceTrackerAppState extends State<PlaceTrackerApp> {
```

Comment:
You should be able to convert this class to a StatelessWidget, now that AppState no longer has to be stored here.